jwst/guider_cds/guider_cds.py

```diff
@@ -98,6 +129,72 @@ def guider_cds(model):
     return new_model
 
 
+def get_ref_arr(model, imshape, gain_model=None, readnoise_model=None):
+    """
+    Short Summary
+    -------------
+    Create gain and readnoise files from their reference files
+
+    Parameters
+    ----------
+    model: data model
+        input data model, assumed to be of type GuiderRawModel
+
+    imshape: (int, int) tuple
+       shape of 2D image
+
+    gain_model: instance of gain Model or None
+        gain for all pixels
+
+    readnoise_model: instance of readnoise Model or None
+        readnoise for all pixels
+
+    Returns
+    -------
+    gain_arr: ndarray, 2-D, float
+        gain values from the ref file, or default value if ref file unavailable
+
+    readnoise_arr: ndarray, 2-D, float
+        readnoise values from the ref file, or default value if ref file
+        unavailable
+    """
+    # if no gain_model was retrieved, use the default value for all pixels
+    if gain_model is None:  # no ref file, so set all pixel's gain to constant default
+        gain_arr = np.zeros(imshape, dtype=np.float32) + DEFAULT_GAIN
+
+        log.info('The appropriate GAIN reference file could not be retrieved,')
+        log.info('so instead, from the reference file %s', DEFAULT_GAIN_FILE)
+        log.info('all gain values will be set to the mean of the SCI values (%s)', DEFAULT_GAIN)
```

Comment:
Is this log message accurate?
First, specifically the use of `from the reference file %s` does not sound correct.

Second, it implies that there's an appropriate reference file in CRDS but it could not be retrieved, i.e. a CRDS problem. Is that what was meant? I assume what was meant is that there's no reference file in CRDS. If so please correct the wording.